src/main/resources/assets/languagereload/lang/zh_tw.json

```diff
@@ -0,0 +1,15 @@
+{
+  "language.default.tooltip": "Minecraft 預設語言",
+
+  "options.languagereload.title": "語言重新載入選項",

```

Comment:
"Language Reload" is the mod name here; it should not be translated (as "Minecraft" in the first string).